pyvista/themes.py

```diff
@@ -2198,6 +2204,50 @@ def use_ipyvtk(self, value):  # pragma: no cover
         else:
             self.jupyter_backend = 'static'
 
+    @property
+    def split_sharp_edges(self) -> bool:
+        """Set or return splitting sharp edges.
+
+        See :ref:`shading_example` for an example showing split sharp edges.
+
+        Examples
+        --------
+        Enable the splitting of sharp edges globally.
+
+        >>> import pyvista
+        >>> pyvista.global_theme.split_sharp_edges = True  # doctest:+SKIP
+        >>> pyvista.global_theme.split_sharp_edges  # doctest:+SKIP
+        True
```

Comment:
```suggestion
        >>> pyvista.global_theme.split_sharp_edges = True
        >>> pyvista.global_theme.split_sharp_edges
        True
```
Can we remove `doctest:+SKIP` ?

Reply:
Unfortunately not. Please see https://github.com/pyvista/pyvista/pull/2575#discussion_r868766072.